pkg/network/master/metrics/register.go

```diff
@@ -32,10 +32,19 @@ var metricEgressFirewallCount = prometheus.NewGauge(prometheus.GaugeOpts{
 	Help:      "The number of egress firewall policies",
 })
 
+// represents the count of multicast enabled namespaces
+var metricMulticastEnabledNamespaceCount = prometheus.NewGauge(prometheus.GaugeOpts{
+	Namespace: metricSDNNamespace,
+	Subsystem: metricSDNSubsystemController,
+	Name:      "num_multicast_enabled_namespaces",
```

Comment:
not suggesting a change here because youre following existing naming convention, but for your own info here is the prometheus naming guide: https://prometheus.io/docs/practices/naming/